test/jit/test_list_dict.py

```diff
@@ -268,19 +268,27 @@ def foo3():
 
```

Comment:
You can't edit these, these are specifically testing TorchScript's parsing support for dict keyword.